.github/workflows/elixir.yml

```diff
@@ -51,9 +51,11 @@ jobs:
         id: deps-cache
         uses: actions/cache@v2
         with:
-          path: deps
-          key: ${{ runner.os }}-mix-v2-${{ hashFiles('**/mix.lock') }}
-          restore-keys: ${{ runner.os }}-mix-v2-
+          path: |
+            deps
+            _build
+          key: ${{ runner.os }}-mix-v3-${{ hashFiles('.tool-versions') }}-${{ hashFiles('**/mix.lock') }}
```

Comment:
It seems like I needed to add `.tool-versions` here for CI to pass so that when you upgrade it forces the rebuild. 
The error can be seen here:
https://github.com/mbta/document_viewer/actions/runs/11506491854/job/32030566877

```
Error: 19:56:41.796 [error] beam/beam_load.c(583): Error loading function ranch_sup:init/1: op allocate_zero u u:
  please re-compile this module with an Erlang/OTP 27 compiler or update your Erlang/OTP version
```